website_snippet_carousel_product/static/src/js/s_product_carousel_frontend.js

```diff
@@ -51,20 +52,22 @@ odoo.define("website_snippet_carousel_product.s_product_carousel", function (req
                             .find("input[name='product_count']")
                             .val();
                         if (!count) {
-                            self.$target.append(
-                                $("<div/>", {class: "col-md-6 offset-md-3"}).append(
-                                    $("<div/>", {
-                                        class:
-                                            "alert alert-warning" +
-                                            " alert-dismissible text-center",
-                                        text: _t(
-                                            "No products was found." +
-                                            " Make sure you have products" +
-                                            " published on the website."
-                                        ),
-                                    })
-                                )
-                            );
+                            if (!session.is_website_user) {
```

Comment:
This excludes also portal users?

Reply:
No, portal users will see the warning message.

Reply:
They shouldn't. Only users with website group must see it (those that are able to modify the domain).

Reply:
Is this comment attended?

Reply:
no

Reply:
Well, it's important, as most e-commerce users are portal ones, so they will see it anyway.

Reply:
Done, now only users with "website.group_website_designer" group will see warning messages